E2E/pageObjects/game.page.ts

```diff
@@ -8,6 +8,7 @@ const locator = {
   },
   text: {
     username: '#username',
+    linkCopiedToClipboard: '//*[contains(text(), "Link copied to clipboard")]',
```

Comment:
Would it work, if you removed `contains` function and leave only exact match by text?